src/config/dynamic-theme-fixes.config

```diff
@@ -3693,6 +3693,25 @@ article, article section, article footer, .css-1fzo8jw, .css-efdc07, article .cs
 
 ================================
 
+brack.ch
+
+CSS
+body, body > header, .popover, .popover-body {
+    background: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-text) !important;
+}
+.list-group-item, .modal-content {
+    background: var(--darkreader-neutral-background) !important;
+}
+a:not(:hover) {
+    color: var(--darkreader-neutral-text) !important;
+}
+a:hover {
```

Comment:
With changes included above whole a:hover entry to change color when hover is not needed.